tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseManagementHttpClient.java

```diff
@@ -57,6 +65,29 @@ public ReleaseArtifactVersionsResponse GetVersions(String project, List<Artifact
         String response = this.ExecutePostmethod(url, body);
         return new Gson().fromJson(response, ReleaseArtifactVersionsResponse.class);
     }
+
+    public void GetReleaseLogs(String project, String releaseId, BuildListener listener) throws ReleaseManagementException
+    {
+        String url = this.accountUrl + project + "/_apis/release/releases/" + "2" + "/logs?api-version=4.0-preview.2";
+        InputStream inputStream = this.ExecuteGetMethodAsStream(url);
+
+        try {
+            FileOutputStream out = new FileOutputStream("/home/chenyl/tmp.zip");
```

Comment:
please remove this home path. You need use File.CreateTempFile.

Reply:
Sorry, I forgot to clean some debugging codes.